### DISCUSSION TITLE
سورس کد اصلی hiddifypanel

### DISCUSSION THREAD

**Discussion by amir1376** [hiddify/Hiddify-Manager#634] [Q&A سوالات رایج]
درود من در پروژه گیتهاب متوجه شدم که اکثر لایبرری های استفاده شده در این پروژه بصورت جدا نصب میشوند
برای مثال hiddifypanel که یک ماژول برای پایتون pip هست از طریق shell نصب شده سورس کد اصلیش رو از کجا میتونیم بخونیم ؟

**Comment by radiumatic** [comment]
@HiddifyOfficial

**Comment by wlfvpn** [comment]
@hiddify1 I looked into the source code and looks like hiddifypanel package is closed source. We would appreciate it you can opensource it.

I reached this message in the hiddifypanel package files, Haha :D 

> #واقعا برای 5 دلار میخوای کرک میکنی؟ حاجی ارزش وقت خودت بیشتره
#Email hiddify@gmail.com for free premium version. Do not crack this cheap product

The problem is not $5 though. I understand to monotize it. There should be other ways of monetizing it while keeping the files opensource. Something like Activation keys in a DB or something like that.
